clustersmgmt/v1/cluster_api_type.go

```diff
@@ -55,6 +57,29 @@ func (o *ClusterAPI) GetURL() (value string, ok bool) {
 	return
 }
 
+// Listening returns the value of the 'listening' attribute, or
+// the zero value of the type if the attribute doesn't have a value.
+//
+// The listening method of the API server.
+func (o *ClusterAPI) Listening() ListeningMethod {
+	if o != nil && o.listening != nil {
+		return *o.listening
+	}
+	return ListeningMethod("")
```

Comment:
I am not familiar with the ocm-sdk logic, but is that what we usually do? No default to one of the methods?
If it were a bool, we would probably default to false.

Reply:
it's a generated code by `make generate`. The defaults will be handled in clusters-service code